src/auditor/instruction_set.jl

```diff
@@ -1177,18 +80,19 @@ whether the object file being pointed to is a 32-bit or 64-bit one:
 """
 function minimum_instruction_set(counts::Dict, is_64bit::Bool)
     if is_64bit
-        if counts["avx512"] > 0
+        if counts["PKU"] + counts["RDSEED"] + counts["ADCX"] + counts["CLFLUSH"] + counts["XSAVEC"] + counts["XSAVES"] + counts["CLWB"] + counts["AVX512EVEX"] + counts["AVX512VEX"] > 0
```

Comment:
```suggestion
        avx512_instruction_categories = ("PKU", "RDSEED", "ADCX", "CLFLUSH", "XSAVEC", "XSAVES", "CLWB", "AVX512EVEX", "AVEX512VEX")
        if any(get.(Ref(counts), avx512_instruction_categories, 0) .> 0)
```

Reply:
I think this form would be a little clearer; and this way I don't have to wonder if the counts can be negative or whatever.